src/main/java/com/textmessenger/service/UserServiceImpl.java

```diff
@@ -75,4 +75,21 @@ public void addLikers(Post post, User user) {
   public List<User> findUsersBySearch(String str) {
     return userRepository.findUsersByEmailOrLogin(str, str);
   }
+
+  @Override
+  public List<User> getFollowings(Long id) {
+    List<User> following = userRepository.getOne(id).getFollowing();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Immediately return this expression instead of assigning it to the temporary variable "following". ([squid:S1488](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1488))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=130&issues=AWU91xBsW2AOP0CkQ2mP&open=AWU91xBsW2AOP0CkQ2mP)

[ISSUE_KEY]: AWU91xBsW2AOP0CkQ2mP